scripts/main.js

```diff
@@ -13,8 +13,12 @@ images[images.length - 1].src = "images/House 2.png";
 images.push(new Image());
 images[images.length - 1].src = "images/House 3.png";
 
+
+
 images[0].onload = function() {
-    c.drawImage(images[0], 5, 5, 40, 40);
+    for (i = 0; i < images.length; i++) {
+        c.drawImage(images[0], i + 5 + 50*i, 5, 40, 40);
```

Comment:
Nice start :smile: You should think about what the value of the second argument, `i + 5 + 50*i`, will be as `i` loops from `0` to `3`.